<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>路径设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/loadtest.png</normaloff>images/loadtest.png</iconset>
  </property>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonInput">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库ip地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonInput_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonInput_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sqlIp">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>数据库ip地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_username">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_password">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
